examples/federated_statistics/README.md

```diff
@@ -0,0 +1,47 @@
+# Objective
+Federated Statistics will provide pre-built-in NVFLARE federated statistics operators ( controller and executors) that 
+generate global statistics based on local client side statistics.
+
+At each clide site, we could have one more datasets (such as "train" and "test" datasets); each datasets may have many 
+features. For each named dataset, for each feature in the dataset, we will calculate the statistics and combined to produce 
+global statistics for the combined features. 
```

Comment:
At each client site, we could have one or more datasets (such as "train" and "test" datasets); each dataset may have many 
features. For each named dataset, for each feature in the dataset, we will calculate the statistics and combine them to produce 
global statistics for the combined features.